### ISSUE TITLE
Game statistics also for scenario

### ISSUE THREAD

**Issue by Halbarad1** [Sh0wdown/UnofficialCrusaderPatch#648]
Hi, 
it would be nice if we could get statistics also for a scenario, so that you can see how much troops you recruited, gold was made, etc. The same way as it works in normal games vs AI/ normal players.

Hi,
es wäre cool wenn man nach einem gespielten Szenario auch die Statistiken angezeigt bekommen würde, nicht nur im Kreuzugmarsch/ bei Online-Spielen. Viele Szenarien beinhalten ja sogar KI-Gegner und funktionieren in etwa genauso wie normale Spiele.

**Comment by J-T-de** [comment]
I am working on it. Sh0wdown named a function (which I can confirm) cUI_RenderEndScore, and in a function before it decides whether or not the statistics overlay gets rendered. Depending on how it is done, it can be easy, or not,,,

**Comment by J-T-de** [comment]
Okay, I kinda dumped some time into this and found out some stuff:

- one needs to hardwire the beginning of `4DC500` (Crusader 1.4.1) so that `4DC560` (Sh0wdown called this one cUI_RenderEndScore) gets always called. This is not enough to render the statistics of scenarios (this only displays the overlay greatest lord), so one also needs to work something out within `4DC560`
- furthermore, the icons to sort stuff get rendered somewhere later in `4DC500`, so this also needs to be investigated (noping out the call to `4DC560` still displays these icons after a skirmish match).

TL;DR: super quick fix failed, stuff needs to be adjusted at multiple different locations. Should be possible, but I will not look into this in the near future.
